packages/api/package.json

```diff
@@ -45,11 +45,11 @@
   "author": "Cumulus Authors",
   "license": "Apache-2.0",
   "dependencies": {
-    "@cumulus/cmrjs": "1.12.1",
-    "@cumulus/common": "1.12.1",
-    "@cumulus/ingest": "1.12.1",
+    "@cumulus/cmrjs": "1.13.0",
+    "@cumulus/common": "1.13.0",
+    "@cumulus/ingest": "1.13.0",
     "@cumulus/logger": "1.11.0",
```

Comment:
Are we not updating the versions for logger? Here and in `api/ecs/async-operation/package.json`